README.rst

```diff
@@ -2,3 +2,139 @@
  REANA example - CMS reconstruction
 ====================================
 
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-cms-reco.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-alice-cms-reco/master/LICENSE
+
+
+About
+======
+This REANA reproducible analysis example demonstrates the reconstruction
+procedure of the CMS collaboration from `raw data <http://opendata.cern.ch/search?page=1&size=20&experiment=CMS&file_type=raw>`_
+to `Analysis Object Data (AOD) <https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookDataFormats#AoD>`_,
+for the year `2011` and the data set `DoubleElectron`.
+
+The workflow consists of the steps need for the samples reconstruction, as taken
+from the `CMS legacy validation repo <https://github.com/cms-legacydata-validation/RAWToAODValidation/tree/master>`_.
+
+Reconstruction procedure
+=========================
+
+1 & 2. Input data and Analysis code
+------------------------------------
+
+Any raw input data from the `CERN open data platform <http://opendata.cern.ch/search?page=1&size=20&experiment=CMS&type=Dataset&subtype=Collision&subtype=Derived&subtype=Simulated&file_type=raw>`_
+should be valid for reconstruction. In this example, the input is taken from:
+`root://eospublic.cern.ch//eos/opendata/cms/Run2011A/DoubleElectron/RAW/v1/000/160/433/C046161E-0D4E-E011-BCBA-0030487CD906.root`
+
+The reconstruction step can be repeated with a configuration file that depends
+on the analyzed data, e.g. `this example <http://opendata.cern.ch/record/43>`_,
+or by creating our own configuration file (created in a CMS VM) and then
+changing the script accordingly:
+
+.. code-block:: console
+
+    cmsDriver.py reco -s RAW2DIGI,L1Reco,RECO,USER:EventFilter/HcalRawToDigi/hcallaserhbhehffilter2012_cff.hcallLaser2012Filter --data --conditions FT_R_53_LV5::All --eventcontent AOD --customise Configuration/DataProcessing/RecoTLR.customisePrompt --no_exec --python reco_cmsdriver2011.py
```

Comment:
If this is `console`, you should put starting prompt (`$ `).  If it would have been `bash` or `script`, then there would be no graphical representation of prompt, only script code.